src/cascadia/TerminalApp/defaults.json

```diff
@@ -1,56 +1,94 @@
+    // Tab UI
+    "alwaysShowTabs": true,
     "showTabsInTitlebar": true,
     "showTerminalTitleInTitlebar": true,
     "tabWidthMode": "equal",
-    "snapToGridOnResize": true,
-    "wordDelimiters": " /\\()\"'-.,:;<>~!@#$%^&*|+=[]{}~?\u2502",
+
+    // Miscellaneous
     "confirmCloseAllTabs": true,
+    "theme": "system",
+    "rowsToScroll": "system",
+    "snapToGridOnResize": true,
 
     "profiles":
-    [
-        {
-            "guid": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
-            "name": "Windows PowerShell",
-            "commandline": "powershell.exe",
-            "hidden": false,
-            "startingDirectory": "%USERPROFILE%",
-            "closeOnExit": "graceful",
-            "colorScheme": "Campbell Powershell",
-            "cursorShape": "bar",
-            "fontFace": "Cascadia Code",
-            "fontSize": 12,
+    {
+        "defaults": {
+            // Console Settings
             "historySize": 9001,
-            "icon": "ms-appx:///ProfileIcons/{61c54bbd-c2c6-5271-96e7-009a87ff44bf}.png",
-            "padding": "8, 8, 8, 8",
             "snapOnInput": true,
-            "useAcrylic": false,
-            "antialiasingMode": "grayscale"
-        },
-        {
-            "guid": "{0caa0dad-35be-5f56-a8ff-afceeeaa6101}",
-            "name": "cmd",
-            "commandline": "cmd.exe",
-            "hidden": false,
+            "padding": "8, 8, 8, 8",
             "startingDirectory": "%USERPROFILE%",
-            "closeOnExit": "graceful",
+
+            // Acrylic Settings
+            "useAcrylic": false,
+            "acrylicOpacity": 0.5,
+
+            // Background Image
+            "backgroundImage": null,
+            "backgroundImageAlignment": "center",
+            "backgroundImageOpacity": 1.0,
+            "backgroundImageStretchMode": "uniformToFill",
+
+            // Color Scheme
             "colorScheme": "Campbell",
+            "foreground": null,
+            "background": null,
+            "selectionBackground": null,
+
+            // Cursor
+            "cursorColor": null,
+            "cursorHeight": null,
             "cursorShape": "bar",
+
+            // Font
             "fontFace": "Cascadia Code",
             "fontSize": 12,
-            "historySize": 9001,
-            "icon": "ms-appx:///ProfileIcons/{0caa0dad-35be-5f56-a8ff-afceeeaa6101}.png",
-            "padding": "8, 8, 8, 8",
-            "snapOnInput": true,
-            "useAcrylic": false,
-            "antialiasingMode": "grayscale"
-        }
-    ],
+
+            // Tab and Dropdown UI
+            "icon": null,
+            "suppressApplicationTitle": false,
+            "tabTitle": null,
+
+            // Miscellaneous
+            "antialiasingMode": "grayscale",
+            "scrollbarState": "hidden",
+            "closeOnExit": "graceful",
+            "hidden": false,
+            "source": null,
+
+            // Experimental Settings
+            "experimental.retroTerminalEffect": false

```

Comment:
so we're going with the "don't make any code changes, and hope that these match the compiled-in defaults forever" option?

Reply:
nope. Latest commit removed these. Unfortunately, if we have profiles as an object instead of a list, I think we just lose all of the profiles.defaults info. So, I've turned it back into a list.

Not happy about this change :/ but I'm still brainstorming a way to make it look prettier.